api/types.go

```diff
@@ -132,6 +134,10 @@ type ListResponse[T any] struct {
 	PaginationResponse `json:",inline"`
 	Count              int `json:"count"`
 	Items              []T `json:"items"`
+
+	// LastUpdateIndex is set to the latest update index for any request made
+	// with the updateIndex query parameter.
+	LastUpdateIndex `json:"-"`
```

Comment:
I'm going to look at moving this out of `ListResponse` again. I tried once, but I ran into problems that I now can't remember.